cmds/core/hexdump/hexdump_test.go

```diff
@@ -65,14 +64,22 @@ func TestHexdump(t *testing.T) {
 			var writeBuf = &bytes.Buffer{}
 			readBuf.WriteString(tt.readInput)
 			if got := hexdump(tt.filenames, readBuf, writeBuf); got != nil {
-				if got.Error() != tt.wantErr.Error() {
-					t.Errorf("hexdump() = '%v', want: '%v'", got, tt.wantErr)
+				// Different Go compilers deliver
+				// different errors: either syscall.ENOENT
```

Comment:
oof, that's unfortunate. Is there a bug opened (with tinygo I presume) to fix that?

Reply:
I'm planning to. I need to see if they're good with a change. It's a very different toolchain ...